K8s/manifest/collectorconfig.yaml

```diff
@@ -0,0 +1,143 @@
+---
```

Comment:
We should not need this anymore. We are deploying the collector with the correct configuration directly in the OpenTelemetryCollector resource